### PR TITLE
Office 365 - iteration 2 Compliance tags

### PR DESCRIPTION
**Pull request by NicoGuini**
|Related issue|
|---|
|https://github.com/wazuh/wazuh/issues/8999|

## Description
2nd interation of Office 365 integration. Adding compliance tags for HIPAA and PCI

### CODE REVIEW

**Review by jcruzlp — APPROVED** [review]
LGTM!

File: ruleset/rules/0755-office365_rules.xml [hunk 1]
```diff
@@ -1140,39 +1141,39 @@ ID: 91531 - 91650
...
+        <group>Exchange,hipaa_164.312.a.1,hipaa_164.312.b,pci_dss_5.3,pci_dss_10.6.1,pci_dss_11.4,pci_dss_11.5</group>
     </rule>
 
     <rule id="91722" level="12">
         <if_sid>91532</if_sid>
         <field name="office365.Operation" type="osregex">^QuarantineRelease$</field>
         <description>Office 365: Released quarantine message</description>
-        <options>no_full_log</options>
-        <group>Exchange</group>
+        <options>no_full_log,pci_dss_10.2.2</options>
```
Comment (jcruzlp) [inline ruleset/rules/0755-office365_rules.xml:1167]: The compliance tag is on the options and it's making the checks fail

**Review by jcruzlp — CHANGES_REQUESTED** [review]

File: ruleset/rules/0755-office365_rules.xml (hunk 1 above)
Comment (NicoGuini) [reply ruleset/rules/0755-office365_rules.xml]: solved

**Review by jcruzlp — APPROVED** [review]

**Review by 72nomada — APPROVED** [review]
lgtm

**Pull request merged**